src/js/experimental/media-chrome-listbox.js

```diff
@@ -12,8 +12,8 @@ template.innerHTML = `
     gap: 0.5em;
     margin: 0;
     padding: 0.5em;
-    background-color: var(--media-listbox-background, var(--media-control-background, rgba(10,10,15, .8)));
-    color: var(--media-text-color, white);
+    background: var(--media-control-background, var(--media-secondary-color, rgb(20 20 30 / 0.7)));
```

Comment:
- Replacing `--media-listbox-background` with generic `--media-control-background` that conforms with other components
  - This includes changing the default value in the composition to be consistent with other components
  - Component-specific styling can be achieved by, e.g.:
```css
media-chrome-listbox {
   --media-control-background: rgb(10 10 15 / 0.8);
}
```
- May want to consider applying this background to the `media-chrome-listbox-item` instead, though this would require some additional changes to the default CSS

Reply:
Changing the colors here changes the contrast ratios of the colors and text between the components. It might be fine, but we'd need to survey the contrast ratios.

Reply:
Is it worth adding a ~ `darker()` to the shared color in a situation like that? We ultimately _want_ people to change these colors, and not have it then be a game of whackamole, so if there's a middle ground somewhere in that it's worth finding.

Reply:
Assuming that these colors aren't good enough, we could add a second set of `-darker` color varilables.

At some point soon, oklch will be in all browsers which would make it a lot easier to do these kinds of things ad-hoc, too.